### ISSUE TITLE
Feature Request: Ability to specify formatter via Rake task or config file

### ISSUE THREAD

**Issue by shepmaster** [brigade/scss-lint#570]
For our setup, we'd like to use the checkstyle formatter in our CI environment. The `RakeTask` class doesn't allow anything more than a set of files to scan and a config file. As a workaround, we are invoking `scss-lint` via a `sh` command:

```ruby
    task :scss_lint do
      sh "scss-lint " \
         "--require 'scss_lint_reporter_checkstyle' " \
         "--format 'Checkstyle' " \
         "--out '#{scss_lint_file}'"
    end
```

**Comment by sds** [comment]
Hey @shepmaster, 

Would happily accept a pull request for this feature!

**Comment by sds** [comment]
Implemented by #573.
